cmd/rook/ceph/cleanup.go

```diff
@@ -87,3 +106,37 @@ func startCleanUp(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func startSubVolumeGroupsCleanUp(cmd *cobra.Command, args []string) error {
```

Comment:
`startSubVolumeGroupsCleanUp` to `startSubVolumeGroupCleanUp` as we are working with single group here.